episodes/episodes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Episodes
+teaching: 5
+exercises: 25
+---
+
+At the end of the previous section,
+you defined the learning objectives for your lesson as a whole.
+
+Rather than write the lesson as a single, long document,
+we recommend that you break it up into chunks
+like chapters in a book or episodes in a season of a TV series.
+This can help manage learners' cognitive load by 
+ensuring that you organise content into coherent, more self-contained chunks,
+and makes it easier for instructors to schedule regular and frequent breaks while teaching.
+Thinking about how content can be broken down like this
+early in the lesson design process
+helps you to consider the path learners will take to reach your defined objectives,
+and identify the component skills and knowledge they will need to pick up on the way.
+
+In The Carpentries, we refer to these individual parts of a lesson as _episodes_,
+to encourage lesson developers to think about them as 
+self-contained units (with their own narrative arc) 
+that nevertheless contribute to the larger whole
+(the theme or story that runs through a full season).
+It also helps to think about the typical length of an episode:
+these chunks should not take more than an hour to finish.
+
+Each episode will exist as a page in the website we will build for our lessons.
+
+## Planning Your Episodes
+
+Before we can start creating the episodes of a lesson,
+we need to spend some time planning out the number and order of them.
+The learning objectives you defined for your lesson can help with this:
+at the very least, it is probably a good idea to have one episode dedicated to each objective.
+You might also find that you can "decompose" the lesson-level objectives 
+into more finely-grained steps that learners can take towards those end points:
+
+- What new knowledge and skills will learners need to acquire to be ready to fulfil the overall objectives for your lesson?
+- What order should these concepts and skills be introduced in? Are some dependent on others?
+- If some of these concepts and skills are complex, can they be broken down even further?
+
+:::::::::::::::::::::::::::::::::::::::: challenge
+
+## Exercise: Defining Episodes for a Lesson (25 minutes)
+
+1. Based on the lesson-level objectives and your knowledge of the lesson topic,
+   divide the lesson up into logical blocks, that should each take no more than 60 minutes to teach.
+2. Next, assign responsibility for one of these episodes to each collaborator in your team.
+   They will focus on this episode for the rest of this training, 
+   and you will teach these episodes in a trial run between parts 1 and 2 of this training.
```

Comment:
I think the ambiguity is ok here, because we will be more thorough and precise later when the focus is on the trial runs.

(For the record, I think we concluded that the team should teach the trial run and everyone should have a role in that even if they do not all take the role of instructor.)